src/System.Net.WebSockets.Client/tests/ClientWebSocketTest.cs

```diff
@@ -21,31 +20,23 @@ public class ClientWebSocketTest
 
         private readonly ITestOutputHelper _output;
 
+        private static bool WebSocketsSupported { get { return WebSocketHelper.WebSocketsSupported; } }
```

Comment:
nit: ClientWebSocketUnitTest.cs has this static property `WebSocketsSupported` defined after the class constructor. So, it would be nice to have the same ordering in this class.